app-backend/db/src/test/scala/com/azavea/rf/database/CampaignDaoSpec.scala

```diff
@@ -468,4 +468,54 @@ class CampaignDaoSpec
       )
     }
   }
+
+  test("get users who have campaign copies") {
+    check {
+      forAll(
+        (
+            userCreates: List[User.Create],
+            userCreate: User.Create,
+            campaignCreate: Campaign.Create,
+            annotationProjectCreate: AnnotationProject.Create
+        ) => {
+          val copyIO = for {
+            parent <- UserDao.create(userCreate)
+            children <- userCreates traverse { u =>
+              UserDao.create(u)
+            }
+            insertedCampaign <- CampaignDao
+              .insertCampaign(
+                campaignCreate.copy(parentCampaignId = None),
+                parent
+              )
+            insertedProject <- AnnotationProjectDao
+              .insert(
+                annotationProjectCreate.copy(
+                  campaignId = Some(insertedCampaign.id),
+                  status = AnnotationProjectStatus.Waiting
+                ),
+                parent
+              )
+            _ <- AnnotationProjectDao.update(
+              insertedProject.toProject
+                .copy(status = AnnotationProjectStatus.Ready),
+              insertedProject.id
+            )
+            _ <- children traverse { child =>
+              CampaignDao.copyCampaign(insertedCampaign.id, child)
+            }
+            cloneOwners <- CampaignDao.getCloneOwners(insertedCampaign.id)
+          } yield cloneOwners
+
+          val cloneOwners = copyIO.transact(xa).unsafeRunSync
+
+          assert(
+            cloneOwners.length == userCreates.length,
```

Comment:
you have a little bit more information to work with than this -- for instance, you can ensure that the returned user ids are the same as those you inserted